literature.bib

```diff
@@ -2452,6 +2504,17 @@ @inproceedings{FLP+:VaMoS21
 	year = 2021
 }
 
+@article{RSL+:SoSyM21,
+	author = {Kamil Rosiak and Alexander Schlie and Lukas Linsbauer and Birgit Vogel{-}Heuser and Ina Schaefer},
+	title = {{Custom-Tailored Clone Detection for IEC 61131-3 Programming Languages}},
+	journal = SoSyM,
+	publisher = Elsevier,
+	volume = {182},
+	pages = {111070},
```

Comment:
this doesn't look right

Reply:
Elsevier strikes once again. Thats the official value and I agree, it does not look right. I will remove the comment and add a comment that the page numbers could not be identified.